data/hai/hai reveal 4 middle.txt

```diff
@@ -879,30 +879,30 @@ mission "Hai Reveal [A07] Deliver the Relays"
+			`Stopping in to <origin>, it takes practically no time to find Terry. She seems to be in motion at almost all times, trailed by a plethora of people constantly bringing things to her attention and then scurrying away once they've received whatever answer they were seeking. Between seeing her and crossing the eighty or so meters to come face to face, you see her put her thumbprint to at least three separate datapads; presumably approvals of some sort.`
+			`	"Ah, <last>. Excellent," she says as you arrive. "Come with me."`
+			`	It turns out that the main concourse of this station, set up as it is for a movie production, is littered with small meeting rooms squeezed in between more significant operations. They'd probably be storage cupboards on a normal station, but at present pretty much everything and everyone on the station seems to be occupied and in use. One diligent attendant blocks the door as she closes it behind you both, and you see the queue of people scatter through the door's glass window. No doubt they'd return as soon as she was available again.`
 			`	"This David Cohen character is an eccentric one, but things are progressing rapidly and smoothly so far. The station is working, and the cover is gaining traction. From what I hear they're even actually making the movie now. They've almost made enough progress that people could believe the movie has been a work in progress for long enough to justify all this."`
 			choice
 				`	"You certainly seem busy."`
 					goto busy
 				`	"Looks like it's thanks to you."`
 			`	She momentarily adopts a self-satisfied smile.`
 			label busy
-			`	"Yes well, this poses a few issues of its own with the increased demand on the relay bandwidth from here, but I wouldn't have my regular job if something like this was too much to stay on top of."`
+			`	"Yes, well, this poses a few issues of its own with the increased demand on the relay bandwidth from here, but I wouldn't have my regular job if something like this was too much to stay on top of."`
 			choice 
 				`	"Your regular job?"`
 					goto regular
 				`	"Your job as a secretary?"`
-			`	She rolls her eyes shakes her head with a little huff. "Ridiculous that that's what they tried to have me doing. No, I mean the job I had right before the end of the war, and the job I still seem to be doing seventy percent of even while I'm here."`
+			`	She rolls her eyes and shakes her head with a little huff. "Ridiculous, that. It's what Sam tried to have me doing. No, I mean the job I had at the Syndicate right before the end of the war; the job I still seem to be doing seventy percent of even while I'm here."`
```

Comment:
Yeah, rereading it I can see what you mean. But, I couldn't tell the first time, and I think others wouldn't have been able to either.

Reply:
Yea, I didn't read it spoken like this either at first, I figured it was a double print on the word.

People DO speak this way, but it's not at all good form in writing. I'm really not sure of a way to make it more clearly read as it would be spoken.

I'd suggest leaving this dangling for now, see if one of the other reviewers like @W1zrad has a better grasp on the technical form?

Reply:
I've changed it to something similar to the original but I don't have much to make it look better. It's still a bit clunky.
>She rolls her eyes and shakes her head with a little huff. "It's ridiculous that Sam tried to make me look like some common witless secretary. No, I mean the job I had right before the end of the war, and the job I still seem to be doing seventy percent of even while I'm here."